Portfolio.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header class="header">
+        <a id = "myName" href="index.html">David Gonzalez</a>
+    </header>
+    <main class="main">
+        <div class="Portfolio-list">
+            <h1>Portfolio</h1>
+            <ul>
+                <li class="project1">
+                    <h3>Project 1</h3>
+                    <img src="placeholder.jpg" alt="project1">
+                    <ul class="project1-descriptor">
+                        <li>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</li>
+                        <li>Quam elementum pulvinar etiam non quam lacus suspendisse faucibus. Tristique et egestas quis ipsum suspendisse.</li>
+                        <li>Imperdiet massa tincidunt nunc pulvinar sapien et ligula ullamcorper malesuada.</li>
+                        <li>Nunc scelerisque viverra mauris in aliquam sem fringilla ut. At quis risus sed vulputate.</li>
+                        <li>Diam maecenas ultricies mi eget. Tellus in metus vulputate eu scelerisque felis imperdiet proin fermentum.</li>
+                    </ul>
+                </li>
+
+
```

Comment:
nitpick: make sure to remove extra blank lines in this file. 1 blank line to separate blocks (like you've done in other files) is ok